### ISSUE TITLE
default tree adapter doesn't seem to be exported via export map

### ISSUE THREAD

**Issue by 43081j** [inikulin/parse5#619]
trying to use it in the p5 tools package:

```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './dist/tree-adapters/default.js' is not defined by "exports" in /parse5-tools/node_modules/parse5/package.json
```

trying to import `parse5/dist/tree-adapters/default.js`.

is it because we have `./dist/index.js` in our export map but nothing else? so we need `dist/tree-adapters/*` too maybe?

cc @fb55

**Comment by fb55** [comment]
I was trying to avoid deep exports, as it is difficult to update relevant paths for CJS and ESM. Ways of accessing the default adapter were discussed in #555.

**Comment by 43081j** [comment]
we actually want the exported values in this case though, not only the types.

as we want to make use of the default tree adapter's functions. so im not sure how else we can get that other than exposing the default tree adapter module.

do you have another way in mind?

**Comment by 43081j** [comment]
@fb55 happy to have a go at doing a PR for this but im not 100% sure what the right export map is...

if i try something like:

```json
    "exports": {
        ".": {
            "import": "./dist/index.js",
            "require": "./dist/cjs/index.js"
        },
        "./dist/tree-adapters/default.js": {
            "import": "./dist/tree-adapters/default.js"
        }
    },
```

we're also forced to add anything the default tree adapter depends on (e.g. `dist/common/html`). it which point we're edging on just exposing everything 🤷‍♂️ 

if you can help me out knowing what the right solution is here i can try write it up

**Comment by wooorm** [comment]
Why not export it from the main module?

**Comment by 43081j** [comment]
@wooorm that would make life a lot easier! ill open a PR, no clue why i didn't consider that :D

**Comment by 43081j** [comment]
This is fixed by #631 i think, the rest of the symbols were already exported top level
